proxy/test/CommunityLocker.ts

```diff
@@ -107,11 +107,25 @@ describe("CommunityLocker", () => {
     });
 
     it("should set time limit per message", async () => {
-        await communityLocker.setTimeLimitPerMessage(0)
+        expect(BigNumber.from(await communityLocker.timeLimitPerMessage(mainnetHash)).toString()).to.be.equal(BigNumber.from(300).toString());
```

Comment:
With `ethereum-waffle` this bulky syntax can be replaced with
```
expect(await communityLocker.timeLimitPerMessage(mainnetHash)).to.be.equal(300);
```

Reply:
Agree